js_modules/dagster-ui/packages/ui-core/src/workspace/VirtualizedRepoAssetTable.tsx

```diff
@@ -202,21 +202,21 @@ const validateExpandedKeys = (parsed: unknown) => (Array.isArray(parsed) ? parse
 /**
  * Use localStorage to persist the expanded/collapsed visual state of asset groups.
  */
-const useAssetGroupExpansionState = (storageKey: string) => {
+export const usePersistedExpansionState = (storageKey: string) => {
```

Comment:
Nice! I think this could even be moved out into a new file so that it's completely separate from anything specific to repos or assets.

Reply:
sounds good, i moved this to a new file